.github/workflows/argo-rollouts-fork-ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: Argo Rollouts Fork CI 
```

Comment:
We should probably have this in a distinct PR, for this specific repo.

The goal of this job is to be able to build rollouts on a fork, we should be able to apply it independently on any branch to contribute to argocd

Reply:
I'll create this in a different PR.

Reply:
Done.